build.gradle.kts

```diff
@@ -4,14 +4,18 @@
  */
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 import com.android.build.gradle.BaseExtension
+import kotlinx.validation.ApiValidationExtension
 
 buildscript {
     repositories {
         google()
         jcenter()
+        // For binary compatibility validator.
+        maven { url = uri("https://kotlin.bintray.com/kotlinx") }
```

Comment:
Can we also add an exclusion clause for the main project to this import? I don't want to think about what will happen to our F-Droid builds otherwise...

Reply:
We can't. At this rate the simplest solution would be to just extract the library out to its own repo and make this repo depend on a Jitpack artifact.

Reply:
I see. I would say we can merge this as is now and explore that option if F-Droid builds should start failing.

Reply:
But there seems an issue with the cache action, do you know what's up with that?

Reply:
Yeah I just force-pushed a fixed version, should pass now.